src/components/Tutorial/index.jsx

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Row, Col } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.scss';
-import {Link} from 'react-router-dom'
+import { Link } from 'react-router-dom';
+import QuestionTabsSection from './QuestionsTab';
 
 const TutorialSection = () => {
+  const [activeRow, setActiveRow] = useState('user-site');
+
+  const handleRowChange = (rowId) => {
+    setActiveRow(rowId);
+  };
+
   return (
-    <section>
+    <section className='w-80'>
```

Comment:
Do not give width within component, you giving widht of the component in the parent, where component is shown.

Reply:
I'll make sure to address this concern by ensuring that width specifications are handled with more care